packages/preview/kdl-unofficial-template/0.1.0/typst.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "kdl-unofficial-template"
+version = "0.1.0"
+authors = ["Martin Heuschober"]
+license = "MIT"
+entrypoint = "lib/lib.typ"
+description = "An unofficial template for community scenarios for the TTRPG 'KULT: divinity lost'"
+category = "book"
```

Categories should always be a list, even if there is a single one.

```suggestion
categories = ["book"]
```